spec/lib/freedom_patches/rspec_mocks_from_described_class_spec.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "RSpec Mocks from_described_class" do
```

Comment:
```suggestion
RSpec.describe "RSpec Mocks from_described_class" do
```

no need for manual require